config/production.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```

Comment:
Not a big deal since this was auto-generated for you, but we should probably remove the placeholder `FOO` in `config/production.js` and `config/staging.js` since we won't really be using those.